src/storage/src/hummock/local_version.rs

```diff
@@ -106,6 +106,10 @@ impl LocalVersion {
             pinned_version,
         }
     }
+
+    pub fn clear_shared_buffer(&mut self) {
```

Comment:
Do we need some `stop` mark to avoid any other `sync` or `ingest_batch` request to hummock?

Reply:
The compute node is not destroyed so when the recovery process finishes, it is still possible to `sync` or `ingest_batch`. But I do notice a potential issue of this PR when writing a test: we need to clear pending writes as well as buffer tracker states too.